### PR TITLE
test pull request 

### PR DESCRIPTION
**Pull request by Dandoo1212**
TEST

### CODE REVIEW

File: src/main/java/Main/Main.java [hunk 1]
```diff
@@ -42,5 +42,7 @@ public static void main(String[] args) throws InterruptedException {
         }
 
         System.out.println("TEST");
+        System.out.println("TEST2");
+        System.out.println("TEST 3 ");
```
Comment (Dandoo1212) [inline src/main/java/Main/Main.java:46]: This is violating SOLID.

File: src/main/java/Main/Main.java [hunk 2]
```diff
@@ -42,5 +42,7 @@ public static void main(String[] args) throws InterruptedException {
         }
 
         System.out.println("TEST");
+        System.out.println("TEST2");
```
Comment (Dandoo1212) [inline src/main/java/Main/Main.java:45]: This is also violating SOLID

**Review by Dandoo1212 — COMMENTED** [review]
do some changes

File: src/main/java/Main/Main.java (hunk 1 above)
Comment (sloczynskikamil) [reply src/main/java/Main/Main.java:46]: Single responsibility
Open–closed
Liskov substitution
Interface segregation
Dependency inversion

**Pull request merged**